pkg/build/api/v1/types.go

```diff
@@ -371,6 +375,13 @@ type SourceBuildStrategy struct {
 	ForcePull bool `json:"forcePull,omitempty" description:"forces the source build to pull the image if true"`
 }
 
+// BuildPostCommitSpec holds a build post commit hook specification.
+type BuildPostCommitSpec struct {
+	// RunBash holds a bash script to be executed in a container running the
+	// build output image.
+	RunBash string `json:"runBash,omitempty" description:"bash script to be executed in a container running the build output image"`
```

Comment:
our conclusion was that this assumes you have bash in your image. RunShellCommand can stand for any type of shell and it might be misleading hence the runBash